src/map/unit.c

```diff
@@ -488,102 +489,142 @@ static int unit_walktoxy_timer(int tid, int64 tick, int id, intptr_t data)
 	return 0;
 }
 
+/**
+ * Timer for delayed execution of unit->walktoxy once triggered
+ * @param tid: Timer ID, unused
+ * @param tick: Tick, unused
+ * @param id: ID of block_list to execute the action
+ * @param data: uint32 data cast to intptr_t with x-coord in lowest 16 bits and y-coord in highest 16 bits
+ * @return 0: success, 1: failure
+ */
 static int unit_delay_walktoxy_timer(int tid, int64 tick, int id, intptr_t data)
```

Comment:
please change the function signature as haru mentioned in his review.

Reply:
Done locally.